notebooks/Notebook3.md

```diff
@@ -18,17 +20,31 @@ exciting milestones, preliminary results, etc. Feel free to include images
 code, and any other resources that you think will help clearly convey your
 design process.
 
+My main goal for this week was to get sbt working on Codespaces. However, I got sick and couldn't make it to class or any of the tutoring hours this week and still couldn't figue out how to set it up on my own. Thankfully, I figured out a way to code through a work sheet which automatically displays the output for each line without having to compile. Next week it will still be important to get sbt working. 
+
```

Comment:
We can work on this during office hours on Tuesday or class on Wednesday, if you want!